### ISSUE TITLE
Offline access of google drive

### ISSUE THREAD

**Issue by bioinfolabmu** [astrada/google-drive-ocamlfuse#838]
Hello!

This morning I installed google-drive-ocamlfuse successfully.  Thank you very much for this nice tool. 

I did see my local google drive, which is in synchronization with my online google drive.  My question is that I purposely turned off the internet to see, whether my local google drive can be used. For example, I tried to open an existing file.  Unfortunately, it does not work. 

One important purpose for me to have a local computer-version of google drive is that in case something bad happens for online version, I have a backup with a local version.  Am I able to have the local access of the local version of google drive?  Am I missing something here?  

Your help will be greatly appreciated

Mu Liang

**Comment by astrada** [comment]
No, sorry, it isn't possible. You should use rclone or a similar tool.

**Comment by bioinfolabmu** [comment]
I see. Thank you for your quick response.  So, Google Drive in the local computer will not actually occupies the same spaces as the one on cloud.  Only the header information is stored locally (small space is required). Am I right?

**Comment by ravenlost** [comment]
Just as a suggestion, I personally always used google-drive-ocamlfuse, but on a daily basis, I just do an rsync from my /mnt/GoogleMount to /mnt/GoogleMount_BACKUP  as means to make sure I always have a local copy (we can never trust enough "free" cloud services!).
